### PR TITLE
Describe the returns attributes creation process

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request describes the returns attributes creation process. 
This block was described due to the fact that it is partially different from creating custom attributes.

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- yes

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- yes

## Affected documentation pages

https://docs.magento.com/user-guide/stores/attributes-returns.html?itm_source=merchdocs&itm_medium=search_page&itm_campaign=federated_search&itm_term=Returns%20Attributes

### CODE REVIEW

File: src/stores/attributes-returns.md [hunk 1]
```diff
@@ -15,7 +15,55 @@ _Returns Attributes_
...
+
+    - None
+    - Alphanumeric
+    - Alphanumeric with Space
+    - Numeric Only
+    - Alpha Only
+    - URL
+    - Email
+
+1. To limit the size of Text Field and Text Area input types, enter the **Minimum Text Length** and **Maximum Text Length**.
```
Comment (leslietilling) [inline src/stores/attributes-returns.md:48]:
```suggestion
1. For the Text Field and Text Area input types, enter the **Minimum Text Length** and **Maximum Text Length**.
```

File: src/stores/attributes-returns.md [hunk 2]
```diff
@@ -15,7 +15,55 @@ _Returns Attributes_
...
+
+1. To apply a preprocessing filter set Input/Output Filter to one of the following:
```
Comment (leslietilling) [inline src/stores/attributes-returns.md:50]:
```suggestion
1. To apply a preprocessing filter, set **Input/Output Filter** to one of the following:
```

File: src/stores/attributes-returns.md [hunk 3]
```diff
@@ -15,7 +15,55 @@ _Returns Attributes_
...
+    - Strip HTML Tags
+    - Escape  HTML Entities
+
+1. To make the attribute visible to customers, set **Show on Storefront** to Yes in the **Storefront Properties** block.
+
+1. For **Sort Order**, enter a number to determine the rating with other attributes.
+
+### Manage the Labels/Options
+
+1. In the left panel, choose Manage Labels/Options.
```
Comment (leslietilling) [inline src/stores/attributes-returns.md:62]:
```suggestion
1. In the left panel, choose **Manage Labels/Options**.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Minor edits

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the pull request successfully imported.

**Pull request merged**